### PR TITLE
Improve developer policies page

### PR DESCRIPTION
**Pull request by philoserf**
- copyedit

### CODE REVIEW

**Review by sigrunixia — APPROVED** [review]

**Comment by sigrunixia** [comment]
Hi @philoserf. I'll go ahead and merge this in. Thanks for your help. :)

**Comment by philoserf** [comment]
Oops. Sorry. I added to the PR mid-flow. Sorry.

File: en/Developer policies.md [hunk 1]
```diff
@@ -10,46 +10,46 @@ Plugins and themes must not:
...
+- Comply with the original licenses of any code your plugin or theme uses, including attribution in the README if required.
 - Respect Obsidian's trademark policy. Don't use the "Obsidian" trademark in a way that could confuse users into thinking your plugin or theme is a first-party creation.
 
 ## Reporting violations
 
-If you encounter a plugin or theme that violates the policies above, please let the developer know by opening a GitHub issue in their repository. Kindly check existing issues to see if it’s already reported.
+If you encounter a plugin or theme that violates the policies above, please inform the developer by opening a GitHub issue in their repository. Kindly check existing issues to see if it’s already been reported.
 
-If the developer doesn’t response after 7 days, [contact the Obsidian team](https://help.obsidian.md/Help+and+support#Report+a+security+issue). For serious violations, you can contact our team immediately.
+[Contact the Obsidian team](https://help.obsidian.md/Help+and+support#Report+a+security+issue) if the developer doesn’t respond after seven days. For serious violations, you can contact our team immediately.
```
Comment (joethei) [inline en/Developer policies.md:41]: This reads as if there are two options, contacting the developer, or contacting us.

We would prefer it if users try to contact the developer first, and only then contact us.

File: en/Developer policies.md [hunk 2]
```diff
@@ -10,46 +10,46 @@ Plugins and themes must not:
...
 - Payment is required for full access.
 - An account is required for full access.
 - Network use. Clearly explain which remote services are used and why they're needed.
 - Accessing files outside of Obsidian vaults. Clearly explain why this is needed.
-- Static ads such as banners and pop-up messages within the plugin's own interface.
-- Server-side telemetry. Link to a privacy policy that explains how the data is handled must be included.
-- Close sourced code. This will be handled on a case by case basis.
+- Static ads, including banners and pop-up messages within the plugin's interface.
+- Server-side telemetry. A link to a privacy policy that explains how the data is handled must be included.
+- Close-sourced code. This will be handled on a case-by-case basis.
```
Comment (joethei) [inline en/Developer policies.md:27]: "Closed-source code" would be better.
(yes, there was a typo in the original)

**Review by joethei — CHANGES_REQUESTED** [review]

**Pull request merged**